src/AzSK.ADO/Framework/BugLog/AutoBugLog.ps1

```diff
@@ -233,8 +233,7 @@ class AutoBugLog : EventBase  {
                     }
                 }
                 else {
-                    $AssignedTo = $metaProviderObj.GetAssignee($ControlResults[0], $this.InvocationContext);
-                    $serviceId = $metaProviderObj.ServiceId;
+                    $AssignedTo = $ControlResults[0].ResourceContext.ResourceOwner
                 }                
             }
             else {
```

Comment:
No need to fetch service ID now. Please remove this. Also bugs are not getting logged because we are checking for not null service ID. Please remove this check